.github/workflows/docs-publish.yml

```diff
@@ -71,7 +72,7 @@ jobs:
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
[Security scanning](https://github.com/TBD54566975/web5-js/security/code-scanning) checks for pinned dependencies using hash so pin to a v4 using the commit hash of the GitHub action.

> For GitHub workflows used in building and releasing your project, pin dependencies by hash.